app/javascript/vite/application.js

```diff
@@ -33,6 +33,11 @@ setupSentry()
 // On each page load, check for old SCP caches, delete any found
 clearOldServiceWorkerCaches()
 
+// Close tooltips; fixes edge case with Bootstrap-default Popper / Tippy tooltips
+document.addEventListener('click', () => {
```

Comment:
THANK YOU - this has been very annoying when I see it in various places in the UI.